code/Controllers/SilverStripeNavigator.php

```diff
@@ -18,6 +19,9 @@
  * New item types can be defined by extending the {@link SilverStripeNavigatorItem} class,
  * for example the "cmsworkflow" module defines a new "future state" item with a date selector
  * to view embargoed data at a future point in time. So the item doesn't always have to be a simple link.
+ *
+ * Class have been moved from `silverstripe/cms` to `silverstripe/admin` and renamed.
+ * @deprecated 4.13.0 Use SilverStripe\Admin\Navigator\SilverStripeNavigator instead
```

Comment:
```suggestion
 * @deprecated 4.13.0 Will be renamed SilverStripe\Admin\Navigator\SilverStripeNavigator
```
Or "Will be moved to" - I'm not sure which is better, so either will do.
But saying "Use [the new name] instead" is misleading since that's not available in CMS 4.

Please make the same change for the deprecation notice itself, and for all of the other classes.

Reply:
I would prefer "Will be moved to" since the (short) name of the class stays, in this case only the namespace changes.

Reply:
DONE

Reply:
Still need to update the message in all of the deprecation notices, as well. The deprecation notice should have the same message as the deprecation annotation.